### ISSUE TITLE
[Error]: The setting name "Screen resolution" is misleading

### ISSUE THREAD

**Issue by LC-Zorg** [OpenTTD/OpenTTD#9734]
### Version of OpenTTD

12.1 and earlier, possibly from 0.6.0-beta1 (r10704)

### Expected result

This setting changes the window size, not the screen resolution, so it should be called "Screen / Window size" or "Gameplay screen / window size".

### Actual result

The title of the option "Screen resolution" suggests that it changes the screen resolution, which is not true. This is confusing and misleading as to the actual function of the setting. Report #9717 is one confirmation of this issue. I also thought that was the way it worked.

### Steps to reproduce

n/a

**Comment by LC-Zorg** [comment]
Question: is this feature needed at all? The only use I see for it is to take screenshots of a certain resolution, but that's a bit of a weak motivation. I don't see any other reason. If someone wants to play on a smaller window, they can minimize it and choose any size.
In my opinion, if it were to be, it clearly requires a name change. On the other hand, if the goal would be to add an actual resolution setting function in the near future in place of the one that resizes the window, it might be better to keep the wrong name for the time being. Here I would like to point out once again that adding this function would only make sense if it were a multiplier of x2, x3, x4, ... Choosing any resolution other than multiple would blur the game view, so making the function work as is currently visible would be a bad solution.

**Comment by glx22** [comment]
If you enable "full screen" it will actually change the screen resolution (at least on Windows).

**Comment by Eddi-z** [comment]
we could instead disable the setting if anything other than "full screen" is used

**Comment by LC-Zorg** [comment]
> If you enable "full screen" it will actually change the screen resolution (at least on Windows).

I never used full screen mode and didn't think it actually worked. Man learns all his life :D

> we could instead disable the setting if anything other than "full screen" is used

I think that would be appropriate. But the question is, could this resolution change work also for a windowed game? Then choosing the resolution 1/2, 1/3, 1/4 should give a very good effect without blurring for screens with high resolutions.

**Comment by Eddi-z** [comment]
> I think that would be appropriate. But the question is, could this resolution change work also for a windowed game? Then choosing the resolution 1/2, 1/3, 1/4 should give a very good effect without blurring for screens with high resolutions.

there's a gui zoom setting for that

**Comment by TrueBrain** [comment]
> we could instead disable the setting if anything other than "full screen" is used

I like that solution.

**Comment by orudge** [comment]
Or it could be renamed to "Window size" if it's not full screen? I at least still like to pick a resolution even though I use OpenTTD in a window, rather than dragging the window to an arbitrary size.

**Comment by PeterN** [comment]
This is absolutely standard terminology...

![image](https://user-images.githubusercontent.com/639850/157853338-a840c57d-e776-44e8-8480-6485137fa101.png)
![image](https://user-images.githubusercontent.com/639850/157853754-fc1f45e5-6804-40be-ae0d-2b31deaa5e5d.png)

**Comment by TrueBrain** [comment]
> Or it could be renamed to "Window size" if it's not full screen? I at least still like to pick a resolution even though I use OpenTTD in a window, rather than dragging the window to an arbitrary size.

Fair enough, that is enough to leave it in. What to name it really becomes bikeshedding .. what-ever you do, someone will complaint. So I suggest to just leave it as it is ..

**Comment by 2TallTyler** [comment]
Working as intended, not a bug.
